<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>556</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Format:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxOutput"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListView" name="listView"/>
      </item>
      <item>
       <widget class="VerticalToolBar" name="verticalToolBar" native="true">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="countLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>617</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_AboutQt"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Add"/>
    <addaction name="action_Remove"/>
    <addaction name="action_RemoveSelected"/>
    <addaction name="separator"/>
    <addaction name="action_CopySelected"/>
    <addaction name="action_CopyAll"/>
    <addaction name="separator"/>
    <addaction name="action_SelectAll"/>
    <addaction name="separator"/>
    <addaction name="action_Boolean"/>
    <addaction name="separator"/>
    <addaction name="action_Clear"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Output</string>
    </property>
    <addaction name="action_OutputFormat"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuVIew">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action_Pack"/>
    <addaction name="action_Unpack"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuVIew"/>
   <addaction name="menuOption"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>About &amp;RangeIDConvertor...</string>
   </property>
  </action>
  <action name="action_AboutQt">
   <property name="text">
    <string>About &amp;Qt...</string>
   </property>
  </action>
  <action name="action_SelectAll">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="action_Add">
   <property name="text">
    <string>Add IDs...</string>
   </property>
  </action>
  <action name="action_Remove">
   <property name="text">
    <string>Remove IDs...</string>
   </property>
  </action>
  <action name="action_RemoveSelected">
   <property name="text">
    <string>Remove Selected IDs</string>
   </property>
  </action>
  <action name="action_Clear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="action_CopyAll">
   <property name="text">
    <string>Copy All</string>
   </property>
  </action>
  <action name="action_CopySelected">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="action_OutputFormat">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Output format:</string>
   </property>
  </action>
  <action name="action_Pack">
   <property name="text">
    <string>Pack</string>
   </property>
   <property name="toolTip">
    <string>Pack</string>
   </property>
  </action>
  <action name="action_Unpack">
   <property name="text">
    <string>Unpack</string>
   </property>
   <property name="toolTip">
    <string>Unpack</string>
   </property>
  </action>
  <action name="action_Boolean">
   <property name="text">
    <string>Boolean...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VerticalToolBar</class>
   <extends>QWidget</extends>
   <header>GUI/VerticalToolBar</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
